actions/kexec/v1/pkg/uefiboot/uefiboot.go

```diff
@@ -0,0 +1,93 @@
+package uefiboot
+
+import (
+	"strconv"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/u-root/u-root/pkg/boot"
+	"github.com/u-root/u-root/pkg/boot/uefi"
+)
+
+var (
+	UEFIDebug       string
+	UEFIBase        string
+	UEFISerialAddr  string
+	UEFISerialWidth string
+	UEFISerialHertz string
+	UEFISerialBaud  string
+)
+
+func Boot() {
+	debug, _ := strconv.ParseBool(UEFIDebug)
+
+	// Set default configuration
+
+	var imageBase uint64
+	imageBase = 0x800000
+
+	uefiConfig := uefi.SerialPortConfig{
+		Type:       uefi.SerialPortTypeIO,
+		BaseAddr:   0x3f8,
+		RegWidth:   1,
+		InputHertz: 1843200,
+		Baud:       115200,
+	}
+
+	// Parse environment variables to see if defaults need overwriting
+
+	if UEFIBase != "" {
+		base, err := strconv.ParseUint(UEFIBase, 16, 64)
+		if err == nil {
+			imageBase = base
+		}
+	}
```

Comment:
I realize this code is experimental, but what do you think about documenting these environment variables, and the expectation that some of these values be base 10 or hexadecimal?